db.go

```diff
@@ -13,42 +13,83 @@ const (
 	NullUserId              UserId = 0
 )
 
+// The DB relies on these constants to be these exact value. If their values are changed, things can go pear-shaped.
```

Comment:
Comments like these should rather be precise. You don't need to say that things will go pear-shaped. Rather let the reader make that decision for himself. For example:
`These constants are embedded inside our database (in the table AuthUserStore). They may never change.`